upgrade/upgrade-scripts/upgrade-container

```diff
@@ -443,6 +443,29 @@ function stop() {
 		sleep 1
 	done
 
+	#
+	# Starting the upgrade container would have enabled ip forwarding
+	# on the host (and in turn disabled LRO). Reset these settings
+	# on a best effor basis.
+	#
+	sysctl -w net.ipv4.ip_forward=0 &>/dev/null
+
+	#
+	# Find all the interfaces configured on the host and set
+	# lro on where possible. The sed command retrieves the names of the
+	# interfaces. The name of the virtual interfaces of containers could
+	# have an @ symbol followed by device name in the output of the ethtool
+	# and this needs to be trimmed off.
+	#
+
+	for i in $(ip -br link | sed 's/^\([^ @]\+\).*/\1/'); do
+		fixed=$(ethtool -k "$i" | grep -e large-receive-offload | grep -ie fixed)
+		if [ -z "$fixed" ]; then
+			echo "updating lro setting for nic $i"
+			ethtool -K "$i" lro on &>/dev/null
```

Comment:
Same comment as above, we could add `warn`..

Reply:
Is there any monitoring on Warning? If we fail to update the setting is there anyway we can be notified?

Reply:
Added the warn. @vimalraj-anbarasu no there is no warning. We do not want to block verification, and in extension, the upgrade due to this, and verification today does not have a good mechanism to pass back warnings. This will get recorded in syslog.